dist/pythonlibs/riotctrl_shell/gnrc.py

```diff
@@ -24,7 +24,9 @@ def __init__(self):
                                   r"icmp_seq=(?P<seq>\d+) ttl=(?P<ttl>\d+)"
                                   r"( rssi=(?P<rssi>-?\d+) dBm)?"
                                   r"( time=(?P<rtt>\d+.\d+) ms)?"
-                                  r"(?P<dup> \(DUP\))?")
+                                  r"(?P<dup> \(DUP\))?"
+                                  r"(response truncated by (?P<truncated>\d+) byte"
+                                  r"(response corrupt at offset (?P<corrupted>\d+)")
```

Comment:
Also I believe there are spaces missing at the end or beginning of the respective strings here.